2/Dockerfile.rhel7

```diff
@@ -53,7 +53,7 @@ RUN rm /opt/openshift/base-plugins.txt && \
     mkdir -p /opt/openshift/plugins && \
     # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
     # HPI file created by rpm to /opt/openshift/plugins folder.
-    for FILENAME in /usr/lib64/jenkins/*hpi ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
+    for FILENAME in /usr/lib/jenkins/*hpi ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
```

Comment:
hm. what lead to this change?
@gabemontero has more familiarity w/ why we were doing it in the first place.